magpie-persist/src/main/java/io/openraven/magpie/plugins/persist/AssetsRepo.java

```diff
@@ -16,26 +16,58 @@
 
 package io.openraven.magpie.plugins.persist;
 
-import org.jdbi.v3.core.Jdbi;
-import org.jdbi.v3.postgres.PostgresPlugin;
-import org.jdbi.v3.sqlobject.SqlObjectPlugin;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.cfg.Environment;
+import org.hibernate.service.ServiceRegistry;
+
+import javax.persistence.EntityManager;
+import java.util.Properties;
 
 import static java.lang.String.format;
 
 public class AssetsRepo {
-  private final Jdbi jdbi;
+
+  private static EntityManager entityManager;
 
   public AssetsRepo(PersistConfig config) {
-    String url = format("jdbc:postgresql://%s:%s/%s", config.getHostname(), config.getPort(), config.getDatabaseName());
-    jdbi = Jdbi.create(url, config.getUser(), config.getPassword())
-      .installPlugin(new PostgresPlugin())
-      .installPlugin(new SqlObjectPlugin());
+
+    Configuration configuration = new Configuration();
+
+    Properties settings = new Properties();
+    settings.put(Environment.DRIVER, "org.postgresql.Driver");
+    settings.put(Environment.URL, format("jdbc:postgresql://%s:%s/%s?stringtype=unspecified",
+      config.getHostname(), config.getPort(), config.getDatabaseName()));
+    settings.put(Environment.USER, config.getUser());
+    settings.put(Environment.PASS, config.getPassword());
+    settings.put(Environment.DIALECT, "org.hibernate.dialect.PostgreSQLDialect");
+    settings.put(Environment.SHOW_SQL, "true");
+    settings.put("stringtype", "unspecified");
+    settings.put(Environment.HBM2DDL_AUTO, "update");
```

Comment:
@belosh59 I assume this is here because you've removed Flyway for Schema migrations.  I just want to ensure this is only temporary while you migrate to Sqitch.  `update` is considered dangerous and shouldn't be used outside of dev/test. =)

Reply:
@kickroot yes you are absolutely right.
This will be there only till the moment I will introduce the Sqitch into the proj